winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: Cards in WinUI Kanban control | Syncfusion
+description: This section describes about cards in Syncfusion WinUI Kanban (SfKanban) control, its elements and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Cards in WinUI Kanban Control
+
+Cards in the WinUI Kanban Control provide a visual representation of tasks as they move through different stages of a workflow. Each card corresponds to a task and displays key information, such as the title, description, tags, and status, helping users track progress and manage tasks effectively.
+
+The appearance and content of the cards can be customized using the below properties of `KanbanModel`.
+
+* `Title` - Gets or sets the string value used to display the header of a card in the KanbanModel.
+* `Description` - Gets or sets the string value used to display the description of a card in the KanbanModel.
+* `Category` - Gets or sets the category associated with the card in the KanbanModel.
+* `Assignee` - Gets or sets the assignee associated with the card in the KanbanModel.
+* `Image` - Specifies the image source representing the card's assignee or related entity.
+* `IndicatorColorKey` - Provides unique key with each Kanban card to dynamically assign indicator colors. It is often used in scenarios where cards are categorized or prioritized by color, based on specific key values.
+* `Id` - Used to uniquely identify each Kanban card, allowing for efficient management, tracking, and updates of individual cards within Kanban control.
+* `Tags` - Used to assign multiple tags to a kanban card, which can be used for categorization, filtering, or additional metadata.
+
+{% highlight C# %}
+
+KanbanModel taskDetail = new KanbanModel();
+taskDetail.Title = "UWP Issue";
+taskDetail.Id = "651";
+taskDetail.Description = "Crosshair label template not visible in UWP";
+taskDetail.Category = "Open";
+taskDetail.IndicatorColorKey = "High";
+taskDetail.Tags = new List<string>() { "Bug Fixing" };
+taskDetail.Image = new Image
+{
+    Source = new BitmapImage(new Uri("ms-appx:///Assets/Kanban/People_Circle1.png"))
+};
+
+{% endhighlight %}
+
+Following code snippet is used to define the colors for each key.
```

Comment:
The following code snippet defines the colors for each key.

Reply:
Changed.